README.md

```diff
@@ -630,6 +630,51 @@ Otherwise the client will raise a `HTTPError`:
 </table>
 </details>
 
+## Get all templates
+
+```python
+response = notifications_client.get_all_templates(
+    template_type=None # optional - SMS_TYPE | EMAIL_TYPE
```

Comment:
Will have a stab at updating the version documentation too, seems like a while since that was done